openai-embedding.el

```diff
@@ -32,28 +32,29 @@
 ;;
 ;;; API
 
-(cl-defun openai-embedding-create ( input callback
-                                    &key
-                                    (content-type "application/json")
-                                    (key openai-key)
-                                    org-id
-                                    (model "text-embedding-ada-002")
-                                    (user openai-user))
+(cl-defun openai-embedding-create (input callback
```

Comment:
Can you revert the indentation (and for all other similar changes), I left a space on purpose so they are aligned. ;)